dyne/ukf.py

```diff
@@ -5,6 +5,41 @@
 from ._common import check_input_arrays, check_measurements
 
 
+def _unscented_transform(x, P, Q, alpha):
```

Comment:
I think the signature should not contain `x` - it can be easily added outside of the function, also see the next point.

Arguments `P` and `Q` should be replaced by a generalized argument `Ps` -- diagonal blocks of a full covariance matrix.